hw_diag/templates/diagnostics_page.html

```diff
@@ -92,38 +66,46 @@ <h3 class="text-center mb-4">Diagnostics Breakdown</h3>
                 </tr>
                 <tr>
                   <td>Region Plan</td>
-                  {% if diagnostics.RE == 'UN123' %}
-                    <td class="text-right">Awaiting Location Assertion</td>
+                  {% if diagnostics.miner_region %}
+                    <td class="text-right">{{ diagnostics.miner_region }}</td>
                   {% else %}
```

Comment:
I would never expect the else to be reached, since miner_region will always have some value. Same goes for other diagnostics using the same pattern.

Reply:
fixed by exposing DiagnosticsReport instead of simple dict.